File: recruitment-automation-system-ui/src/app/services/models/user-detail-response.ts
/* tslint:disable */
/* eslint-disable */
import { User } from '../models/user';
export interface UserDetailResponse {
  address?: string;
  email?: string;
  firstName?: string;
  id?: number;
  lastName?: string;
  phoneNumber?: string;
  user?: User;
  username?: string;
}

File: recruitment-automation-system/src/main/java/com/RAS/recruitment_automation_system/usersDetail/UserDetailResponse.java
package com.RAS.recruitment_automation_system.usersDetail;

import com.RAS.recruitment_automation_system.user.User;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UserDetailResponse {
    private Integer id;
    private String username;
    private String email;
    private String firstName;
    private String lastName;
    private String phoneNumber;
    private String address;
}
